app/notify_client/service_api_client.py

```diff
@@ -163,7 +163,7 @@ def remove_user_from_service(self, service_id, user_id):
         return self.delete(endpoint, data)
 
     @cache.delete('service-{service_id}-templates')
-    def create_service_template(self, name, type_, content, service_id, subject=None, process_type='normal',
+    def create_service_template(self, name, type_, content, service_id, subject=None,
```

Comment:
This is weird, if this wasn't flowing through given that it has a default value of `normal` right in the function signature!

I wonder if the method below (`update_service_template` on line 192 is the culprit, or at least part of the issue, since that specifically has `None` as the default `process_type` argument.  It shouldn't be given the `process_type` check on line 209, but it may not hurt to be absolutely sure in that spot, too.

Reply:
Addressed with subsequent updates!